### PR TITLE
Update PLCrashReporter to include privacy manifest

### PR DESCRIPTION
**Pull request by AnatolyPristensky**
Things to consider before you submit the PR:

* [x] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests?
* [ ] Did you check UI tests on the sample app? They are not executed on CI.
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Add privacy manifest

## Related PRs or issues

[User Story 103999](https://dev.azure.com/msmobilecenter/Mobile-Center/_workitems/edit/103999): [plcrashreporter] Apple Privacy Manifest  #294

### CODE REVIEW

File: PrivacyInfo.xcprivacy [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>NSPrivacyAccessedAPITypes</key>
```
Comment (WWSellers) [inline PrivacyInfo.xcprivacy:5]: Should the file include entries for 'NSPrivacyTracking' and 'NSPrivacyTrackingDomains'?
(probably setting NSPrivacyTracking to 'false' and NSPrivacyTrackingDomains to an empty array)

Comment (AnatolyPristensky) [reply PrivacyInfo.xcprivacy:9]: Added

**Review by DmitriyKirakosyan — APPROVED** [review]

File: PrivacyInfo.xcprivacy [hunk 2]
```diff
@@ -2,19 +2,26 @@
...
+	<key>NSPrivacyTrackingDomains</key>
+	<array/>
 	<key>NSPrivacyAccessedAPITypes</key>
+	<array/>
+	<key>NSPrivacyCollectedDataTypes</key>
 	<array>
 		<dict>
-			<key>NSPrivacyAccessedAPITypeReasons</key>
+			<key>NSPrivacyCollectedDataType</key>
+			<string></string>
```
Comment (WWSellers) [inline PrivacyInfo.xcprivacy:15]: Was the intention to remove this entry?  Both the `NSPrivacyCollectedDataType` and `NSPrivacyCollectedDataTypePurposes` contain empty strings.

File: PrivacyInfo.xcprivacy [hunk 3]
```diff
@@ -27,6 +34,18 @@
...
+		<dict>
+			<key>NSPrivacyCollectedDataType</key>
+			<string>NSPrivacyCollectedDataTypeOtherDiagnosticData</string>
+			<key>NSPrivacyCollectedDataTypeLinked</key>
+			<false/>
+			<key>NSPrivacyCollectedDataTypeTracking</key>
+			<false/>
+			<key>NSPrivacyCollectedDataTypePurposes</key>
+			<array>
+				<string></string>
```
Comment (WWSellers) [inline PrivacyInfo.xcprivacy:46]: Should this contain a reason string? 
Perhaps NSPrivacyCollectedDataTypePurposeAppFunctionality

**Review by lucen-ms — APPROVED** [review]

**Pull request merged**